core/nuxeo-operation.js

```diff
@@ -228,6 +238,18 @@ import './nuxeo-page-provider.js';
           options.headers[`enrichers-${type}`] = v;
```

Comment:
not related at all but I see that nuxeo-operaiton is importing nuxeo-page-provider element and I don't see a reason why.
Would you mind cleaning that up using your ticket?

Reply:
yep, of course ;)

Reply:
i made some checks before removing this import and i see the code [here](https://github.com/nuxeo/nuxeo-elements/blob/f8e865124142c5305f63b557df7395bd18930209/core/nuxeo-operation.js#L199) there is a check on the `input` if it's type is a PageProvider 
perhaps i miss some thing but i think the import is required here for this check.

Reply:
Indeed, it is required there.

Reply:
👍